packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -502,15 +493,15 @@ function resolve(
       if (inst == null || inst.render == null) {
         child = inst;
         validateRenderResult(child, Component);
-        continue;
+        return;
```

Comment:
Are these equivalent? What was happening before, when the loop continued to another iteration?

Reply:
isValidElement would return false and the loop would break. returning from the function here is equivalent to continue before, since there's nothing in the loop body after this function call.

Reply:
👍, thanks!